src/client/layouts/shared/Consents.tsx

```diff
@@ -1,23 +1,21 @@
 import React, { FunctionComponent } from 'react';
```

Comment:
the changes in this file are from cleaning up the `EnhancedConsentTest` code, and adding in shared code/new header too